### PR TITLE
Create Hyderabad-India-CDKGlobal.json

### PR DESCRIPTION
**Pull request by deepeshpatel**
For GDCR-18 at CDK Global, Hyderabad, India

### CODE REVIEW

File: _data/events_gdcr2018/Hyderabad-India-CDKGlobal.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "title": "Coderetreat @ CDK Global (India) Pvt. Ltd.",
+  "url": "https://sites.google.com/view/cdkhyderabadevents",
+  "moderators": [
+    "Vibhor.Mamgain@cdk.com",
```
Comment (rradczewski) [inline _data/events_gdcr2018/Hyderabad-India-CDKGlobal.json]: [Same remark](https://github.com/coderetreat/coderetreat.github.io/pull/366#discussion_r224328090) as in #366 - would be nice to either use names (nicknames or abbreviated names are fine) or twitter handles. We'll be reaching out to you on Twitter and will promote your event with the moderators mentioned.

**Comment by thodorisbais** [comment]
Welcome @deepeshpatel , have a good one 🎉

File: _data/events_gdcr2018/Hyderabad-India-CDKGlobal.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "title": "Coderetreat @ CDK Global (India) Pvt. Ltd.",
+  "url": "https://sites.google.com/view/cdkhyderabadevents",
+  "moderators": [
+    "Vibhor Mamgain",
+    "Deepesh Patel",
```
Comment (thodorisbais) [inline _data/events_gdcr2018/Hyderabad-India-CDKGlobal.json]: @deepeshpatel can you please also remove the full name reference here?

Comment (thodorisbais) [reply _data/events_gdcr2018/Hyderabad-India-CDKGlobal.json]: and also keep only the actual facilitator here as just done in #375

**Pull request merged**